packages/eslint-plugin-x/package.json

```diff
@@ -42,8 +42,7 @@
     "eslint-plugin-prettier": "^3.1.3",
     "eslint-plugin-tsdoc": "^0.2.5",
     "eslint-plugin-vue": "^6.2.2",
-    "prettier": "^2.1.2",
-    "standard-version": "^8.0.0"
```

Comment:
There is still a reference to standard version in `eslint-plugin-x/lib/index.js`

Reply:
Yes I also see this. I think fixing it and we are ready to merge.